contrib/build-wheel.sh

```
@@ -88,21 +97,233 @@ if [ "$CUDA_MAJOR" -ne 12 ] && [ "$CUDA_MAJOR" -ne 13 ]; then
+try_install_torch() {
+    local PROBE=$1
+    local VER=$2
+    local CHANNEL=$3
+    install_torch "$PROBE" "$VER" "$CHANNEL" >/dev/null 2>&1
+}
+
+# Echo "stable", "nightly", or "unavailable" depending on whether
+# torch==${VER}.* can be installed for this (Python, CUDA, arch) combo.
+torch_classify() {
+    local VER=$1
+    local PROBE="/workspace/venv-probe-py$(slug "$PYTHON_VERSION")-torch$(slug "$VER")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)